docs/docs/technology/liquid-staking/technical-overview.md

```diff
@@ -0,0 +1,58 @@
+# Technical Overview
+
+The primary interaction between Parachains and the Polkadot Relay chain within the Polkadot ecosystem occurs through Cross-Chain Messaging (XCM). From the perspective of users on Composable Polkadot, the main concern is to stake and unstake DOT, without needing to worry about the underlying mechanics. 
+
+The Liquid Staking pallet residing on Composable Polkadot is responsible for executing functions such as bonding, unbonding, and rebonding with the Polkadot Relay chain. The Relay chain updates validators in every Era, and the Liquid Staking pallet processes user requests and operates in sync with the Relay chain's Era updates. This functionality is facilitated by the MatchingPool module.
+
+:::info
+An Era refers to a (whole) number of sessions, which is the period that the validator set and each validator's active nominator set is recalculated and where rewards are paid out. An era equals one day on the Polkadot Relay chain.
+:::
+
+## DOT Distribution
+
+To optimise APY and risk management, the Liquid Staking pallet utilises multiple derivative accounts on the Polkadot Relay chain, each of which acts as an independent stash account. These accounts are used to distribute funds effectively based on a sophisticated fund allocation algorithm. 
```

Comment:
not clear security implications of this

Reply:
afaik chain has one account only.  not sure it can create more

Reply:
unless there is such feature added to derive accou ts on behalf of parachain(afaik composable runtime has such for dao chain)

Reply:
@dzmitry-lahoda it uses utility_pallet 
UtilityCall
https://github.com/ComposableFi/composable/blob/liquid-staking/code/parachain/frame/pallet-xcm-helper/src/lib.rs#L330-L349

first it move assets from sovereign account to derivative account.
then bond/nominate/unstake from this derivative account.